saleor/graphql/order/mutations/order_confirm.py

```diff
@@ -91,11 +92,13 @@ def perform_mutation(cls, root, info, **data):
                 payment,
                 manager,
             )
-        order_confirmed(
-            order,
-            info.context.user,
-            info.context.app,
-            manager,
-            send_confirmation_email=True,
+        transaction.on_commit(
```

Comment:
Can we go through all mutations and make sure that we always use transaction+ transaction.on_commit?
Also, line 87 `order_captured` looks like something that should also be in on_commit

Reply:
Yes - I put `order_captured` into `on_commit`. 
For other mutations - we will make some refactor to reorganise how events are called in them. As for now we want to merge this fix separately, before we work on other mutations.